web/admin/src/App.css

```diff
@@ -36,8 +55,25 @@ ul {
   margin: 0 0 1em 0;
 }
 
+.sectionContainer {
+  border: 1px solid #E3ECF8;
+  border-radius: 5px;
+  margin: 20px;
+  background-color: #FFFFFF;
+  /* box-shadow: 0 2px 4px 0 rgba(170,185,202,0.2); */
+  padding: 0px 25px;
+}
+
+.containerRow {
+  display: flex;
+  flex-direction: nowrap;
```

Comment:
Is `nowrap` a valid value for `flex-direction`?  Do you mean `white-space: nowrap` ?